mlir/include/mlir/Transforms/Passes.h

```diff
@@ -15,6 +15,7 @@
 #define MLIR_TRANSFORMS_PASSES_H
 
 #include "mlir/Pass/Pass.h"
+#include "mlir/Pass/PassManager.h"
```

Comment:
Why is PassManager.h being included now?

Reply:
The inliner pass contains a vector of `OpPassManager` in `InlinerOptions`. After this change, the size of `OpPassManager` needs to be known to compile a TU containing this header.